### PR TITLE
update the doc kubeedge_configure

### PR DESCRIPTION
**Pull request by XJangel**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind documentation

**What this PR does / why we need it**:
update the configuration steps

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by XJangel** [comment]
> Will add the `advertise-address` in this PR? @XJangel

For the deployment of single CloudCore, the default advertise-address is just the IP address of CloudCore. So I didn't mention it here. Maybe we can do it  in the documentation of HA CloudCore.

**Comment by fisherxu** [comment]
> Maybe we can do it in the documentation of HA CloudCore.

The `advertise address` is not only used in ha mode. If edgecore connect to cloudnode through a public(floating) IP, we should set the floating ip in the config file. And the floating Ip can't be  automatically acquired. :)

**Comment by XJangel** [comment]
> > Maybe we can do it in the documentation of HA CloudCore.
> 
> The `advertise address` is not only used in ha mode. If edgecore connect to cloudnode through a public(floating) IP, we should set the floating ip in the config file. And the floating Ip can't be automatically acquired. :)

Get it! I will modify it right now.

**Comment by fisherxu** [comment]
Here you don't need explain ha, as long as you explain all the addresses(like floating ip) exposed to the edge node should be added to `advertise address`.

**Comment by XJangel** [comment]
> Here you don't need explain ha, as long as you explain all the addresses(like floating ip) exposed to the edge node should be added to `advertise address`.

yep~

File: docs/setup/kubeedge_configure.md [hunk 1]
```diff
@@ -74,6 +74,19 @@ In the cloudcore.yaml, modify the below settings.
 2. Before KubeEdge v1.3: check whether the cert files for `modules.cloudhub.tlsCAFile`, `modules.cloudhub.tlsCertFile`,`modules.cloudhub.tlsPrivateKeyFile` exists.
 
     From KubeEdge v1.3: just skip the above check. If you configure the CloudCore certificate manually, you must check if the path of certificate is right. 
+    
+    
+    
+    **Note:** If your KubeEdge version is before the v1.3, then just skip the step 3.
+    
+3. Configure the IP addresses of CloudCore which are exposed to the edge nodes(like floating IP) in the `advertiseAddress`
```
Comment (fisherxu) [inline docs/setup/kubeedge_configure.md:82]:
```suggestion
3. Configure all the IP addresses of CloudCore which are exposed to the edge nodes(like floating IP) in the `advertiseAddress`, which will be add to SANs in cert of cloudcore. 
```

**Pull request merged**